core-bundle/src/Resources/contao/library/Contao/Model.php

```diff
@@ -152,6 +152,12 @@ public function __construct($objResult=null)
 			// Create the related models
 			foreach ($arrRelated as $key=>$row)
 			{
+				if (!isset($this->arrRelations[$key]['table']))
+				{
+					$this->arrRelated[$key] = null;
```

Comment:
Shouldn‘t we throw an exception here instead? Because this case should never occur.

Reply:
I guess both works 🤷 but we're setting the relation to `null` a few lines below as well

Reply:
Yes, but that is an expected case. This one is not, so we should not discard it silently.

Reply:
changed in b40ab81779b7b9b5524a4eb3059c48ab80d69ea9